test/CMakeLists.txt

```diff
@@ -56,10 +56,11 @@ if (SPGLIB_USE_SANITIZER)
 	else ()
 		string(TOLOWER ${SPGLIB_USE_SANITIZER} SPGLIB_USE_SANITIZER)
 		target_compile_options(Spglib_symspg PRIVATE
-				-fsanitize=${SPGLIB_USE_SANITIZER}
+				-fsanitize=${SPGLIB_USE_SANITIZER} -static-libasan
```

Comment:
Note to self: use shared-library sanitizer, but add `LD_PRELOAD` to `find_library(libasan)`: https://github.com/Kitware/VTK/blob/b541e465508aab1f6ad4c863e71d0a422301dda2/CMake/vtkSanitize.cmake#L29